buildpipeline/osx.groovy

```diff
@@ -0,0 +1,76 @@
+@Library('dotnet-ci') _
+
+// Incoming parameters.  Access with "params.<param name>".
+// Config - Build configuration. Note that we don't using 'Configuration' since it's used
+//          in the build scripts and this can cause problems.
+// OuterLoop - If true, runs outerloop, if false runs just innerloop
+
+def submittedHelixJson = null
+
+simpleNode('OSX10.12','latest') {
```

Comment:
@mmitche what is the magic names for osx here? Or is osx still missing from the new pipeline builds?

For a private build log of my attempt see:
https://ci3.dot.net/job/dotnet_corefx/job/master/job/GenPRTest/job/osx-Config_Debug+OuterLoop_false_prtest/3/console